### PR TITLE
fix: change test config and prepare for capability filtering

### PR DESCRIPTION
**Pull request by ia3andy**

### CODE REVIEW

**Comment by fabric8cd** [comment]
This PR was deployed **with mocked data** in the following URLs for your appreciation:
- Frontend: https://fabric8-launcher-frontend-pr-1251-app.surge.sh
- Welcome App: https://fabric8-launcher-frontend-pr-1251-welcome-app.surge.sh
- Storybook: https://fabric8-launcher-frontend-pr-1251-storybook.surge.sh

File: packages/launcher-app/.gitignore [hunk 1]
```diff
@@ -27,4 +27,5 @@ yarn-error.log*
 .idea/
 *.swp
 .vscode/
-*.code-workspace
\ No newline at end of file
+*.code-workspace
+.history
```
Comment (edewit) [inline packages/launcher-app/.gitignore]: why does `launcher-app` has it's own `.gitignore`, the one in the root of the project should be enough?

File: packages/launcher-component/src/flows/__tests__/flow-helpers.tsx [hunk 2]
```diff
@@ -2,14 +2,13 @@ import { act, fireEvent } from 'react-testing-library';
 import { waitForTick } from 'launcher-client';
 
 export async function flushPromises() {
-// FIXME: use the new await version when it's available https://github.com/facebook/react/pull/14853
-  act(() => {
+  await act(async () => {
```
Comment (edewit) [inline packages/launcher-component/src/flows/__tests__/flow-helpers.tsx]: :tada:

File: packages/launcher-app/.gitignore (hunk 1 above)
Comment (ia3andy) [reply packages/launcher-app/.gitignore]: @edewit right that's weird ^^

**Pull request merged**